components/switch/metadata/mods.md

```diff
@@ -1,39 +1,47 @@
-| Modifiable custom properties                        |
-| --------------------------------------------------- |
-| `--mod-animation-duration-100`                      |
-| `--mod-animation-duration-200`                      |
-| `--mod-border-width-200`                            |
-| `--mod-focus-indicator-gap`                         |
-| `--mod-focus-indicator-thickness`                   |
-| `--mod-line-height-100`                             |
-| `--mod-switch-background-color`                     |
-| `--mod-switch-background-color-disabled`            |
-| `--mod-switch-background-color-selected-default`    |
-| `--mod-switch-background-color-selected-disabled`   |
-| `--mod-switch-background-color-selected-down`       |
-| `--mod-switch-background-color-selected-focus`      |
-| `--mod-switch-background-color-selected-hover`      |
-| `--mod-switch-control-height`                       |
-| `--mod-switch-control-label-spacing`                |
-| `--mod-switch-control-width`                        |
-| `--mod-switch-focus-indicator-color`                |
-| `--mod-switch-focus-indicator-thickness`            |
-| `--mod-switch-font-size`                            |
-| `--mod-switch-handle-background-color`              |
-| `--mod-switch-handle-border-color-default`          |
-| `--mod-switch-handle-border-color-disabled`         |
-| `--mod-switch-handle-border-color-down`             |
-| `--mod-switch-handle-border-color-focus`            |
-| `--mod-switch-handle-border-color-hover`            |
-| `--mod-switch-handle-border-color-selected-default` |
-| `--mod-switch-handle-border-color-selected-down`    |
-| `--mod-switch-handle-border-color-selected-focus`   |
-| `--mod-switch-handle-border-color-selected-hover`   |
-| `--mod-switch-height`                               |
-| `--mod-switch-label-color-default`                  |
-| `--mod-switch-label-color-disabled`                 |
-| `--mod-switch-label-color-down`                     |
-| `--mod-switch-label-color-focus`                    |
-| `--mod-switch-label-color-hover`                    |
-| `--mod-switch-spacing-top-to-control`               |
-| `--mod-switch-spacing-top-to-label`                 |
+| Modifiable custom properties                             |
+| -------------------------------------------------------- |
+| `--mod-animation-duration-100`                           |
+| `--mod-animation-duration-200`                           |
+| `--mod-border-width-200`                                 |
+| `--mod-cjk-line-height-100`                              |
+| `--mod-focus-indicator-gap`                              |
+| `--mod-focus-indicator-thickness`                        |
+| `--mod-line-height-100`                                  |
```

Comment:
It looks like a few of these mods referencing "100" and "200" global token names should be renamed in the CSS. For example, something like:
`--mod-animation-duration-100` --> `--mod-switch-animation-duration` 
`--mod-animation-duration-200` --> `--mod-switch-animation-duration-label`

Looking at `--mod-border-width-200`, it looks like it may be useful to have a new `--spectrum-switch-border-width` custom property set to `--spectrum-border-width-200` instead of using it directly in various places.